model/src/test/java/mops/FragebogenTest.java

```diff
@@ -107,4 +108,31 @@ public void loescheFrageByIdTest() {
     Assertions.assertFalse(textFragen.contains(neueFrage));
   }
 
+
+  @Test
+  public void getFrageTest() {
+    TextFrage neueFrage = new TextFrage("lorem");
+    fragebogen.addFrage(neueFrage);
+    TextFrage frage = (TextFrage) fragebogen.getFrage(neueFrage.getId());
+    Assertions.assertEquals(frage.getId(), neueFrage.getId());
```

Comment:
Please compare the whole question objects, not only the IDs.
```suggestion
    Assertions.assertEquals(frage, neueFrage);
```